playbooks/configure-release.yml

```diff
@@ -120,16 +120,25 @@
         force: "{{ osa_force_clone | default(false) }}"
 
     - name: Copy basic files into place
-      command: "cp -Rv /opt/openstack-ansible/etc/openstack_deploy /etc/openstack_deploy"
+      shell: |
+        rsync -av \
+              --include='*.yml' --include='*/' --exclude='*' \
+              "/opt/openstack-ansible/etc/openstack_deploy/" \
```

Comment:
I guess this is fine (hard coded path), since we are being prescriptive with rpc-o